lib/unifex/code_generator/base_types/int.ex

```diff
@@ -24,8 +24,7 @@ defmodule Unifex.CodeGenerator.BaseTypes.Int do
     def generate_arg_serialize(name, _ctx) do
       ~g"""
       ({
-      int #{name}_int = #{name};
-      ei_x_encode_longlong(out_buff, (long long)#{name}_int);
+      ei_x_encode_longlong(out_buff, (long long)#{name});
```

Comment:
this was to make sure that the argument is actually integer, not only something castable to long long